x-pack/plugins/security_solution/public/common/components/toolbar/bulk_actions/use_bulk_alert_assignees_items.tsx

```diff
@@ -151,8 +151,10 @@ export const useBulkAlertAssigneesItems = ({
     [TitleContent, hasIndexWrite, isPlatinumPlus, renderContent]
   );
 
-  return {
-    alertAssigneesItems,
-    alertAssigneesPanels,
-  };
+  return useMemo(() => {
```

Comment:
I have nothing against memoization, but as long as the returned object is destructed in all hook calls:

`const { alertAssigneesItems, alertAssigneesPanels } = useBulkAlertAssigneesItems(...);`

which is, and should always be the case, does this change have any impact?